lib/exe/js.php

```diff
@@ -173,7 +173,7 @@ function js_load($file){
         // is it a include_once?
         if($match[1]){
             $base = utf8_basename($ifile);
-            if($loaded[$base]){
+            if(array_key_exists($base, $loaded) && $loaded[$base] === true){
```

Comment:
Please note that you use here a strict comparison, while it was previous a loose comparison.
http://php.net/manual/en/language.operators.comparison.php
http://php.net/manual/en/types.comparisons.php